doc/tutorial/ranges/index.md

```diff
@@ -263,20 +263,26 @@ What can you learn about the different refinements and their relation to the ran
 
 ## The bitcompressed vector
 
-If you followed the alphabet tutorial closely, you will know that seqan3::dna4 needs only two bits to represent it's
-state.
+If you followed the alphabet tutorial closely, you will know that seqan3::dna4 needs only two bits to represent it's state.
```

Comment:
I am not sure what you are changing here, but if you touch this line, please replace "it's" with "its"

Reply:
I did not change anything there but can fix the typo.